common/src/main/scala/hmda/query/repository/ModifiedLarRepository.scala

```diff
@@ -18,8 +18,21 @@ class ModifiedLarRepository(tableName: String,
     * @param lei
     * @return the number of rows removed
     */
-  def deleteByLei(lei: String): Future[Int] =
-    db.run(sqlu"DELETE FROM #${tableName} WHERE lei = $lei")
+  def msaMds(lei: String, filingYear: Int): Future[Vector[(String, String)]] =
+    db.run{
+      sql"""SELECT DISTINCT msa_md, msa_md_name
+                         FROM modifiedlar2018 WHERE UPPER(lei) = ${lei.toUpperCase} AND filing_year = ${filingYear}"""
+        .as[(String, String)]
+    }
+
+  /**
+    * Deletes entries in the Modified LAR table by their LEI
+    * @param lei
+    * @return the number of rows removed
+    */
+  def deleteByLei(lei: String, filingYear: Int): Future[Int] =
+    db.run(
+      sqlu"DELETE FROM #${tableName} WHERE lei = $lei and filing_year = $filingYear")
```

Comment:
shouldnt the LEI references  here be wrapped in UPPER? @omnipresent07

Reply:
Fixed.